packages/docs-reanimated/scripts/build-og-images.jsx

```diff
@@ -57,9 +69,25 @@ async function buildOGImages() {
   const imageBuffer = fs.readFileSync(imagePath);
   const base64Image = `data:image/png;base64,${imageBuffer.toString('base64')}`;
 
-  dirs.map(async ({ dir, files }) => {
-    files.map(async (file) => {
-      const header = getMarkdownHeader(path.resolve(baseDirPath, dir, file));
+  const blogFiles = await fs.promises.readdir(blogDirPath);
+  const blogFilesFiltered = blogFiles
+    .filter((file) => file.endsWith('.md') || file.endsWith('.mdx'))
+    .map((file) => ({
+      dir: '',
+      files: [file],
+    }));
```

Comment:
Why we can obtains docs files in the same way as blog files? Is it possible to unify implementation?